location/OBARegionHelper.m

```diff
@@ -63,8 +65,7 @@ - (void)setNearestRegion {
             OBARegionV2 *region = (OBARegionV2 *)obj;
             BOOL showExperimentalRegions = NO;
 
-            if ([[NSUserDefaults standardUserDefaults] boolForKey:@"kOBAShowExperimentalRegionsDefaultsKey"]) showExperimentalRegions = [[NSUserDefaults standardUserDefaults]
-                                                                                                                                         boolForKey:@"kOBAShowExperimentalRegionsDefaultsKey"];
+            if ([[NSUserDefaults standardUserDefaults] boolForKey:@"kOBAShowExperimentalRegionsDefaultsKey"]) showExperimentalRegions = [[NSUserDefaults standardUserDefaults] boolForKey:@"kOBAShowExperimentalRegionsDefaultsKey"];
```

Comment:
I'll look at tackling this separately simply because it is unrelated to the purpose of this commit and needs to be replaced in a number of locations.